ols/src/llms/llm_loader.py

```diff
@@ -331,14 +330,10 @@ def _watson_llm_instance(self) -> LLM:
 
         # WatsonX uses different parameter names
         llm_model = Model(
-            model_id=self.llm_params.get(
-                "model_id", os.environ.get("WATSON_MODEL", None)
-            ),
+            model_id=self.llm_params.get("model_id", None),
             credentials=creds,
             params=params,
-            project_id=self.llm_params.get(
-                "project_id", os.environ.get("WATSON_PROJECT_ID", None)
-            ),
+            project_id=self.llm_params.get("project_id", None),
```

Comment:
we're going to have to sort this out when we start implementing watson support, but ultimately it needs to come from the config, it can't come from llm_params.